### PR TITLE
UCP: Add support for endpoints using multiple transports.

### LINKED ISSUE #521: osu_iscatter fails while adding an elem to the progress chain
The command line to reproduce the failure is below.
fails on:
```
notifier.c:42   Fatal: overflow in progress chain while adding ucp_stub_ep_progress
```

1. may need to increase the np value to the maximun to make this reproduce faster.
2. can also reproduce with one tl in UCX_TLS.
3. doesn't reproduce on every run but happens often enough.

```
/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/install/bin/mpirun -np 32 -mca btl_openib_warn_default_gid_prefix 0 --bind-to core --tag-output --timestamp-output --display-map -mca pml ucx -x UCX_DEVICES=mlx5_1:1 -x UCX_TLS=all --map-by node /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/tests/osu_micro_benchmark/osu-micro-benchmarks-5.0/mpi/collective/osu_iscatter
 Data for JOB [2224,1] offset 0

 ========================   JOB MAP   ========================

 Data for node: r-hp01	Num slots: 24	Max slots: 0	Num procs: 16
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 0 Bound: socket 0[core 0[hwt 0]]:[B/././././././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 2 Bound: socket 0[core 1[hwt 0]]:[./B/./././././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 4 Bound: socket 0[core 2[hwt 0]]:[././B/././././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 6 Bound: socket 0[core 3[hwt 0]]:[./././B/./././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 8 Bound: socket 0[core 4[hwt 0]]:[././././B/././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 10 Bound: socket 0[core 5[hwt 0]]:[./././././B/./././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 12 Bound: socket 0[core 6[hwt 0]]:[././././././B/././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 14 Bound: socket 0[core 7[hwt 0]]:[./././././././B/./././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 16 Bound: socket 0[core 8[hwt 0]]:[././././././././B/././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 18 Bound: socket 0[core 9[hwt 0]]:[./././././././././B/./.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 20 Bound: socket 0[core 10[hwt 0]]:[././././././././././B/.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 22 Bound: socket 0[core 11[hwt 0]]:[./././././././././././B][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 24 Bound: socket 1[core 12[hwt 0]]:[./././././././././././.][B/././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 26 Bound: socket 1[core 13[hwt 0]]:[./././././././././././.][./B/./././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 28 Bound: socket 1[core 14[hwt 0]]:[./././././././././././.][././B/././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 30 Bound: socket 1[core 15[hwt 0]]:[./././././././././././.][./././B/./././././././.]

 Data for node: r-hp02	Num slots: 24	Max slots: 0	Num procs: 16
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 1 Bound: socket 0[core 0[hwt 0]]:[B/././././././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 3 Bound: socket 0[core 1[hwt 0]]:[./B/./././././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 5 Bound: socket 0[core 2[hwt 0]]:[././B/././././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 7 Bound: socket 0[core 3[hwt 0]]:[./././B/./././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 9 Bound: socket 0[core 4[hwt 0]]:[././././B/././././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 11 Bound: socket 0[core 5[hwt 0]]:[./././././B/./././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 13 Bound: socket 0[core 6[hwt 0]]:[././././././B/././././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 15 Bound: socket 0[core 7[hwt 0]]:[./././././././B/./././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 17 Bound: socket 0[core 8[hwt 0]]:[././././././././B/././.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 19 Bound: socket 0[core 9[hwt 0]]:[./././././././././B/./.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 21 Bound: socket 0[core 10[hwt 0]]:[././././././././././B/.][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 23 Bound: socket 0[core 11[hwt 0]]:[./././././././././././B][./././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 25 Bound: socket 1[core 12[hwt 0]]:[./././././././././././.][B/././././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 27 Bound: socket 1[core 13[hwt 0]]:[./././././././././././.][./B/./././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 29 Bound: socket 1[core 14[hwt 0]]:[./././././././././././.][././B/././././././././.]
 	Process OMPI jobid: [2224,1] App: 0 Process rank: 31 Bound: socket 1[core 15[hwt 0]]:[./././././././././././.][./././B/./././././././.]

 =============================================================Tue Dec 22 03:08:16 2015[1,0]<stdout>:
Tue Dec 22 03:08:16 2015[1,0]<stdout>:# OSU MPI Non-blocking Scatter Latency Test v5.0
Tue Dec 22 03:08:16 2015[1,0]<stdout>:# Overall = Coll. Init + Compute + MPI_Test + MPI_Wait
Tue Dec 22 03:08:16 2015[1,0]<stdout>:
Tue Dec 22 03:08:16 2015[1,0]<stdout>:# Size           Overall(us)       Compute(us)    Pure Comm.(us)        Overlap(%)Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309:0]    notifier.c:42   Fatal: overflow in progress chain while adding ucp_stub_ep_progress
Tue Dec 22 03:08:16 2015[1,0]<stderr>:==== backtrace ====
Tue Dec 22 03:08:16 2015[1,0]<stderr>: 3 0x0000000000036d83 __ucs_abort()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/src/ucs/debug/log.c:223
Tue Dec 22 03:08:16 2015[1,0]<stderr>: 4 0x0000000000033395 ucs_notifier_chain_add()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/src/ucs/datastruct/notifier.c:41Tue Dec 22 03:08:16 2015[1,0]<stderr>: 5 0x000000000000ca3e ucp_stub_pending_add()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/src/ucp/wireup/stub_ep.c:46
Tue Dec 22 03:08:16 2015[1,0]<stderr>: 6 0x000000000000616f uct_ep_pending_add()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/src/uct/api/uct.h:1117
Tue Dec 22 03:08:16 2015[1,0]<stderr>: 7 0x000000000000c6da ucp_tag_send_slow()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/src/ucp/tag/tag_send.c:104
Tue Dec 22 03:08:16 2015[1,0]<stderr>: 8 0x0000000000004d20 mca_pml_ucx_isend()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/mpi-install/a0Qo/src/ompi/ompi/mca/pml/ucx/pml_ucx.c:464
Tue Dec 22 03:08:16 2015[1,0]<stderr>: 9 0x0000000000006767 NBC_Start_round()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/mpi-install/a0Qo/src/ompi/ompi/mca/coll/libnbc/nbc.c:412Tue Dec 22 03:08:16 2015[1,0]<stderr>:10 0x000000000000710b NBC_Start()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/mpi-install/a0Qo/src/ompi/ompi/mca/coll/libnbc/nbc.c:665
Tue Dec 22 03:08:16 2015[1,0]<stderr>:11 0x0000000000020d36 ompi_coll_libnbc_iscatter()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/mpi-install/a0Qo/src/ompi/ompi/mca/coll/libnbc/nbc_iscatter.c:161
Tue Dec 22 03:08:16 2015[1,0]<stderr>:12 0x00000000000aefc7 PMPI_Iscatter()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/mpi-install/a0Qo/src/ompi/ompi/mpi/c/profile/piscatter.c:153
Tue Dec 22 03:08:16 2015[1,0]<stderr>:13 0x0000000000401263 main()  /hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/tests/osu_micro_benchmark/osu-micro-benchmarks-5.0/mpi/collective/osu_iscatter.c:105
Tue Dec 22 03:08:16 2015[1,0]<stderr>:14 0x0000003698c1ed1d __libc_start_main()  ??:0Tue Dec 22 03:08:16 2015[1,0]<stderr>:15 0x0000000000400fb9 _start()  ??:0
Tue Dec 22 03:08:16 2015[1,0]<stderr>:===================Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] *** Process received signal ***
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Signal: Aborted (6)
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Signal code:  (-6)
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 0] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/lib64/libpthread.so.0[0x369900f710]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 1] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/lib64/libc.so.6(gsignal+0x35)[0x3698c32925]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 2] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/lib64/libc.so.6(abort+0x175)[0x3698c34105]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 3] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/install/lib/libucs.so.2(__ucs_abort+0x118)[0x7ffff2f96d88]Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 4] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/install/lib/libucs.so.2(ucs_notifier_chain_add+0xd5)[0x7ffff2f93395]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 5] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/install/lib/libucp.so.2(+0xca3e)[0x7ffff34c3a3e]Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 6] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/install/lib/libucp.so.2(ucp_ep_add_pending+0x3f)[0x7ffff34bd16f]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 7] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/ucx_src/install/lib/libucp.so.2(+0xc6da)[0x7ffff34c36da]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 8] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/install/lib/openmpi/mca_pml_ucx.so(mca_pml_ucx_isend+0x131)[0x7ffff36d0d20]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[ 9] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/install/lib/openmpi/mca_coll_libnbc.so(+0x6767)[0x7ffff18c9767]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[10] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/install/lib/openmpi/mca_coll_libnbc.so(NBC_Start+0x2b)[0x7ffff18ca10b]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[11] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/install/lib/openmpi/mca_coll_libnbc.so(ompi_coll_libnbc_iscatter+0x6a4)[0x7ffff18e3d36]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[12] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/install/lib/libmpi.so.0(MPI_Iscatter+0x4c6)[0x7ffff7d17fc7]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[13] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/tests/osu_micro_benchmark/osu-micro-benchmarks-5.0/mpi/collective/osu_iscatter[0x401263]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[14] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/lib64/libc.so.6(__libc_start_main+0xfd)[0x3698c1ed1d]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] Tue Dec 22 03:08:16 2015[1,0]<stderr>:[15] Tue Dec 22 03:08:16 2015[1,0]<stderr>:/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20151222_023028_45456_90477_r-hp01/installs/qONX/tests/osu_micro_benchmark/osu-micro-benchmarks-5.0/mpi/collective/osu_iscatter[0x400fb9]
Tue Dec 22 03:08:16 2015[1,0]<stderr>:[r-hp01:26309] *** End of error message ***--------------------------------------------------------------------------
mpirun noticed that process rank 0 with PID 0 on node r-hp01 exited on signal 6 (Aborted).
--------------------------------------------------------------------------+ rc=134
+ exit 134


```

### PR DESCRIPTION
**Pull request by yosefe**
See https://github.com/openucx/ucx/wiki/Connection-establishment
Fixes #521 and #497

### CODE REVIEW

**Comment by yosefe** [comment]
@alex-mikheev @shamisp @alinask 
sorry for mega-commit, but it's a big change that had to be done at once otherwise it would not work, hope that the wiki page can be used to make sense of it.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1726/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1122/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1728/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1126/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1730/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1124/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1127/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1731/ for details.

**Comment by shamisp** [comment]
@MattBBaker can you please test on cray.

**Comment by MattBBaker** [comment]
@shamisp it's the return of wireup messages being too big. In one sided wireup there is a message that is 137 bytes long, blowing our limit by 9 bytes.

**Comment by yosefe** [comment]
@MattBBaker is it possible to reduce the size of `uct_sockaddr_smsg_ugni_t`?

**Comment by MattBBaker** [comment]
Maybe. I can try and shred the gni struct that gets sent as part of the address. 
Pinging @hppritcha for wisdom on putting gni_smsg_attr_t on a diet.

**Comment by hppritcha** [comment]
You ought to be able to generate 
```C
msg_type;
buff_size ;
mbox_maxcredit;
msg_maxsize;
```
locally as long as you always use the same parameters for all SMSG mailbox connections.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1129/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1733/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1131/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1735/ for details.

**Comment by MattBBaker** [comment]
@yosefe I merged in the UGNI fix. :+1:

File: src/ucp/wireup/stub_ep.h [hunk 1]
```diff
@@ -8,34 +8,53 @@
...
-#include <ucs/type/class.h>
 
 
 /**
  * Stub endpoint, to hold off send requests until wireup process completes.
+ * It is placed instead UCT endpoint before it's fully connected, and for AM
+ * endpoint it also contains an auxiliary endpoint which can send wireup messages.
+ */
+struct ucp_stub_ep {
+    uct_ep_t            super;         /* Derive from uct_ep */
```
Comment (shamisp) [inline src/ucp/wireup/stub_ep.h]: if you already use doxygen style comments, I would suggest to go all the way and use /**< ... */

File: src/ucs/datastruct/notifier.h [hunk 2]
```diff
@@ -49,6 +50,7 @@ static inline void ucs_notifier_chain_call(ucs_notifier_chain_t *chain)
     ucs_notifier_chain_elem_t *elem;
 
     ucs_notifier_chain_for_each(elem, chain) {
+        ucs_memory_cpu_load_fence();
```
Comment (shamisp) [inline src/ucs/datastruct/notifier.h]: why ?

**Comment by shamisp** [comment]
@yosefe - back to you

File: src/ucs/datastruct/notifier.h (hunk 2 above)
Comment (yosefe) [inline src/ucs/datastruct/notifier.h]: I want to allow adding (but not removing) elements to the call chain from progress thread, while going over the array from main thread.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1742/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1138/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:retest

**Comment by yosefe** [comment]
@shamisp Failure is unrelated

**Comment by shamisp** [comment]
:+1:

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1743/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1139/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1744/ for details.

**Pull request merged**